src/download/download_main.cc

```diff
@@ -424,11 +421,10 @@ DownloadMain::do_peer_exchange() {
     // Create the new m_ut_pex_list by removing any 'removed'
     // addresses from the original list and then adding the new
     // addresses.
-    m_ut_pex_list.erase(std::set_difference(m_ut_pex_list.begin(), m_ut_pex_list.end(), removed.begin(), removed.end(),
-                                            m_ut_pex_list.begin(), SocketAddressCompact_less()), m_ut_pex_list.end());
+    m_ut_pex_list.erase(std::set_difference(m_ut_pex_list.begin(), m_ut_pex_list.end(), removed.begin(), removed.end(), m_ut_pex_list.begin(), SocketAddressCompact_less), m_ut_pex_list.end());
```

Comment:
The `set_difference` calls are so long they should be multiline, with each begin/end and ftor on separate lines.

Reply:
This is the best I could get clang-format to do.